src/app/checks/create/page.tsx

```
@@ -88,13 +89,10 @@ export default async function CreateCheckPage() {
     <CreateCheckStoreProvider>
       <PageHeading title='Create KnowledgeCheck' />
-      <div className='columns-xl gap-12 space-y-12'>
+      <div className='grid grid-cols-1 gap-8 lg:grid-cols-[repeat(auto-fill,minmax(780px,1fr))]'>
         <GeneralSection />
-        <Card disableHoverStyles className='break-inside-avoid'>
-          <h2 className='text-lg'>Settings</h2>
-          <div className='h-[500px]'></div>
-        </Card>
         <QuestionsSection />
+        <SettingsSection />
         <Card className='h-60 break-inside-avoid' disableHoverStyles children={undefined} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)